### PR TITLE
features/Converter/45-convert-box

### PR DESCRIPTION
**Pull request by blueretrograde**
-Changed the styling to a grid for more flexibility across different sizes/screens
-General styling to reflect the Figma designs

https://trello.com/c/tDTVqqmY/45-create-a-currency-conversion-component

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
   "rules": {
     "react/jsx-uses-react": "error",
     "react/react-in-jsx-scope": "off",
-    "react/prop-types": 0
+    "react/prop-types": 0,
+    "no-unused-vars": "off"
```
Comment (kirsty-nology) [inline .eslintrc.json:19]: this needs to be removed

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Dropdown } from 'rsuite';
+import iconMap from "../../utils/currencyIcons";
+import Button from "../Button/Button";
+import altConvert from "../../assets/images/alternate-convert.png";
+import Convert from "../../assets/images/convert-green.png"
+
+import "./CurrencyConverter.scss";
+
+
```
Comment (kirsty-nology) [inline src/components/currencyConverter/CurrencyConverter.jsx:10]: remove extra line

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            setToCurrency(currencyList.find(currency => currency != newFromCurrency));
+        }
+        setFromCurrency(newFromCurrency);
+    };
+    const handleToChange = (eventKey, event) => setToCurrency(event.target.textContent);
+
+    return (
+
+    <div className="convertBox"> 
+        <h3 id="header__amount">Amount</h3>
```
Comment (kirsty-nology) [inline src/components/currencyConverter/CurrencyConverter.jsx:28]: `className=converter__amount`

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        <h3 id="header__from">From</h3>
+        <h3 id="header__to">To</h3>
```
Comment (kirsty-nology) [inline src/components/currencyConverter/CurrencyConverter.jsx:30]: `className=converter__to`

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+        <input  
+            id="input__amount"
```
Comment (kirsty-nology) [inline src/components/currencyConverter/CurrencyConverter.jsx:33]: `className=converter__input`

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+            type="number"
+            placeholder="10.00"
+            onChange={handleAmountChange}
+            value={amount}/>
+            
+        <div className="input__from">
```
Comment (kirsty-nology) [inline src/components/currencyConverter/CurrencyConverter.jsx:39]: `className=converter__source`

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+            <Dropdown  id="from" title={fromCurrency} icon={iconMap[fromCurrency]} onSelect={handleFromChange} value={fromCurrency}>
```
Comment (kirsty-nology) [inline src/components/currencyConverter/CurrencyConverter.jsx:40]: `className=converter__sourceInput`

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+            <img  id="convert" src={Convert} alt="" />
+        </div>
+        
+        <div className="input__to">
+            <Dropdown  id="to" title={toCurrency} icon={iconMap[toCurrency]} onSelect={handleToChange}>
+                {toOptions}
+            </Dropdown>
+        </div>
+
+        <div className="convertBox__button">
```
Comment (kirsty-nology) [inline src/components/currencyConverter/CurrencyConverter.jsx:55]: `className=converter__submitWrapper`

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+            <img src={altConvert} alt="convertLogo" />
+            <Button className="convert__button" buttonText={"Convert"}/>
```
Comment (kirsty-nology) [inline src/components/currencyConverter/CurrencyConverter.jsx:57]: `className=converter__submitButton`

**Review by kirsty-nology — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (blueretrograde) [reply .eslintrc.json:19]: removed

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+        const newFromCurrency = event.target.textContent;
+        if (toCurrency == newFromCurrency) {
+            setToCurrency(currencyList.find(currency => currency != newFromCurrency));
+        }
+        setFromCurrency(newFromCurrency);
+    };
+    const handleToChange = (eventKey, event) => setToCurrency(event.target.textContent);
+
+    return (
+
```
[the remark this answers is not in the document]
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:26]: done

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+    <div className="convertBox">
```
[the remark this answers is not in the document]
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:27]: done

File: src/components/currencyConverter/CurrencyConverter.jsx (hunk 3 above)
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:28]: done

File: src/components/currencyConverter/CurrencyConverter.jsx (hunk 4 above)
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:30]: done

File: src/components/currencyConverter/CurrencyConverter.jsx (hunk 5 above)
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:33]: done

File: src/components/currencyConverter/CurrencyConverter.jsx (hunk 6 above)
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:39]: done

File: src/components/currencyConverter/CurrencyConverter.jsx (hunk 7 above)
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:40]: done

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+            onChange={handleAmountChange}
+            value={amount}/>
+            
+        <div className="input__from">
+            <Dropdown  id="from" title={fromCurrency} icon={iconMap[fromCurrency]} onSelect={handleFromChange} value={fromCurrency}>
+                        {fromOptions}
+            </Dropdown>
+        </div>
+
+        <div className="convertBox__btn">
```
[the remark this answers is not in the document]
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:45]: done

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+            <img  id="convert" src={Convert} alt="" />
```
[the remark this answers is not in the document]
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:46]: done

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+        
+        <div className="input__to">
```
[the remark this answers is not in the document]
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:49]: done

File: src/components/currencyConverter/CurrencyConverter.jsx [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+            <Dropdown  id="to" title={toCurrency} icon={iconMap[toCurrency]} onSelect={handleToChange}>
```
[the remark this answers is not in the document]
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:50]: done

File: src/components/currencyConverter/CurrencyConverter.jsx (hunk 9 above)
Comment (blueretrograde) [reply src/components/currencyConverter/CurrencyConverter.jsx:57]: done

**Review by kirsty-nology — APPROVED** [review]

**Review by stu-slicer — APPROVED** [review]

**Pull request merged**